android/src/main/java/expo/modules/paywallsdk/HeliumPaywallSdkModule.kt

```diff
@@ -142,36 +142,33 @@ class HeliumPaywallSdkModule : Module() {
         NativeModuleManager.currentModule?.sendEvent("onHeliumPaywallEvent", eventMap)
       }
 
-      // Initialize on a coroutine scope
-      CoroutineScope(Dispatchers.Main).launch {
```

Comment:
Only thing that may be concerning is that it's now up to the implementer to make sure this is on the main thread? Not sure if that's a big concern?